modules/xmpp/JingleSessionPC.js

```diff
@@ -56,9 +57,8 @@ function JingleSessionPC(me, sid, peerjid, connection,
     // stable and the ice connection state is connected.
     this.modifySourcesQueue.pause();
 }
-//XXX this is badly broken...
-JingleSessionPC.prototype = JingleSession.prototype;
-JingleSessionPC.prototype.constructor = JingleSessionPC;
+
+util.inherits(JingleSessionPC, JingleSession);
```

Comment:
@tsareg and @hristoterezov, I added Babel + ES2015 in a PR while trying to fix a failing automatic tests. Eventually, I manually rewrote my modifications to ES5 (because it's hard for me to engage in a discussion with the team on the subject of ES6 from Sydney).